swarmrl/losses/proximal_policy_loss.py

```diff
@@ -205,11 +205,10 @@ def compute_loss(self, actor: FlaxModel, critic: FlaxModel, episode_data):
         model_tuple : tuple  FlaxModel, FlaxModel
             The updated actor and critic network.
         """
-        feature_data = episode_data.item().get("features")
-        old_log_probs_data = episode_data.item().get("log_probs")
-        action_data = episode_data.item().get("actions")
-        # will return the reward per particle.
-        reward_data = episode_data.item().get("rewards")
+        old_log_probs_data = episode_data.log_probs
+        feature_data = np.array(episode_data.features)
```

Comment:
does this has to be converted into an array? 
Especially for the graphs, it would be cool if this can stay flexible.

Reply:
No need for an array at all actually, it should come out as one. I will make this change.

Reply:
Before I change out the numpy arrays, which are currently required for the code to work, I think we should wait until we have a case for graphs during which we can adapt the loss to handle this generic pytree input.